sycl/test/basic_tests/accessor/local_accessor_default_ctor.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
I would suggest that we refactor this into a [unit-test](https://github.com/intel/llvm/tree/sycl/sycl/unittests)